modules/apps/collaboration/blogs/blogs-web/src/main/resources/META-INF/resources/blogs/css/main.scss

```diff
@@ -148,6 +148,12 @@
 	}
 
 	.edit-entry {
+		.alloy-editor-placeholder:before {
```

Comment:
That would center every placeholder we have in Blogs, and some of them shouldn't. This way we inherit the parent text-aligment.